ui-components/src/headers/SiteHeader.scss

```diff
@@ -281,32 +281,34 @@
         .navbar-dropdown-container {
           background-color: white;
           position: absolute;
-          margin-left: -124px;
+          margin-left: -144px;
```

Comment:
I think we usually try not to edit ui-components unless this is something thats broken upstream as well and we contribute it back upstream. Is this broken for them or only for us?

If its only for us we should override their css in override.scss

Reply:
I'm not sure if it's broken upstream, as I can't seem to create an account on their dev site. I moved all this stuff into override.scss anyways.